private-templates-service/client/src/utils/requireShared/requireShared.tsx

```diff
@@ -57,7 +57,7 @@ const requireShared = <P extends object>(Component: React.ComponentType<P>) => {
     render() {
       return (
         <React.Fragment>
-          {this.isShared() ?
+          {!this.isShared() ?
```

Comment:
Are you sure about this?

Reply:
nice catch, i was using this for testing.